### ISSUE TITLE
Material file types

### ISSUE THREAD

**Issue by lara0915** [readchina/materials#1]
Hi all,

I would like to raise up the issue on how we regulate the file types for different materials we upload. For instance, for the consent forms, I propose that we scan/take photos of the forms and convert the images to pdf files. There might be similar issues on other file types, sizes of files and resolutions of photos etc. that we need to think about. Shall we discuss the above next week?

**Comment by EveYLin** [comment]
[template_Interview Informed Consent-English.pdf](https://github.com/readchina/materials/files/4992403/template_Interview.Informed.Consent-English.pdf)
[template_interview informed consent-Chinese.pdf](https://github.com/readchina/materials/files/4992405/template_interview.informed.consent-Chinese.pdf)

**Comment by duncdrum** [comment]
Yes these are great, @EveYLin can you create a PR and put them into the folder. We should make two minor changes though, we can discuss them in our meeting later.
